modules/apps/object/object-rest-test/src/testFunctional/macros/ObjectDefinitionAPI.macro

```diff
@@ -460,9 +460,7 @@ definition {
 				-u test@liferay.com:test
 		''';
 
-		var response = JSONCurlUtil.put(${curl});
-
-		return ${response};
+		return ${curl};
```

Comment:
the method is _relatedObjectEntries but we actually don't relate objects here. it's just curl, why?

Reply:
Because the two methods used this curl, one is breakRelationshipBetweenEntries, the other is relateObjectEntries

Reply:
they have the same parameter, but different requests, so I think they can use the same curl.

Reply:
maybe we can change the method name then so it is more generic and accurate to both relate them and delete them?

Reply:
Sure, maybe we can name them "putRelationshipBetweenEntries" and "breakRelationshipBetweenEntries" and the basic curl  name is "_relateObjectEntries", what do you think of it?

Reply:
the thing is that _relateObjectEntries sounds like a call to action to relate the objects, when if fact you would use this curl to break the relationships as well, it is misleading.

Reply:
maybe we can use _curlObjectEntries as the name

Reply:
as we talked about in the meeting `_curlObjectEntriesRelationship` sounds okay :) Thanks for the idea! :)